app/services/stats_cleanup.py

```
@@ -205,45 +156,19 @@ def cleanup_old_records(self, db: Session) -> dict:
-                        ),
-                    ),
-                )
-                .delete(synchronize_session=False)
-            )
-
             db.commit()
 
             logger.info(
-                f"Deleted {records_deleted} old FileRecord entries, {inventory_deleted} missing inventory entries, "
-                f"and {transfers_deleted} remote transfer jobs"
+                f"Deleted {records_deleted} old FileRecord entries, {inventory_deleted} missing inventory entries"
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)